src/main/java/CommunicateCaseHandle.java

```diff
@@ -0,0 +1,134 @@
+import java.util.Scanner;
+public class CommunicateCaseHandle{
+    protected ExceptionsHandle e;
+    public CommunicateCaseHandle(){
+        e = new ExceptionsHandle();
+    }
```

Comment:
add a new line after } separating the next method

Reply:
OK!